src/Files.App/Helpers/CloudDrivesDetector.cs

```diff
@@ -78,6 +78,12 @@ private static Task<IEnumerable<ICloudProvider>> DetectGenericCloudDrive()
 						driveType = appName;
 					}
 
+					// iCloud specific
```

Comment:
Shouldn't this go in the AppleCloud detector file?

Reply:
There's already detection in place for namespace, which is why I removed AppleCloudDetector. If you're referring to the fact that I added code specifically for iCloud, then shouldn't Nextcloud (right above) also have its own detector class?

Reply:
@gave92

Reply:
It's ok, cloud drives that can be detected from `Explorer\Desktop\NameSpace` registry should go in DetectGenericCloudDrive()